satellite/vouchers/service.go

```diff
@@ -49,29 +49,29 @@ func NewService(log *zap.Logger, satellite signing.Signer, cache *overlay.Cache,
 }
 
 // Request receives a voucher request and returns a voucher and an error
-func (service *Service) Request(ctx context.Context, req *pb.VoucherRequest) (_ *pb.Voucher, err error) {
+func (service *Service) Request(ctx context.Context, req *pb.VoucherRequest) (_ *pb.VoucherResponse, err error) {
 	defer mon.Task()(&ctx)(&err)
 
 	peer, err := identity.PeerIdentityFromContext(ctx)
 	if err != nil {
-		return &pb.Voucher{}, Error.Wrap(err)
+		return nil, Error.Wrap(err)
```

Comment:
Why are we returning nil here instead of an empty voucher? Just wondering

Reply:
Not sure if there is a particular reason to choose one over the other

Reply:
Yeah the internet says it doesn't really matter, but then some people say nil is better than a pointer to an empty struct I think because it may be clearer. But in either case, you're fine here!